scripts/start-ssr.js

```diff
@@ -62,7 +62,7 @@ const copyPublicFiles = () => {
 runWebpack(webpackConfig[1])
   .then(copyPublicFiles)
   .then(() => {
-    const url = `http://${hosts[0]}:${port.backend}`;
+    const url = `http://${hosts[0]}:${port.backend}${initialPath}`;
```

Comment:
Should this use a [url joining library](https://www.npmjs.com/package/url-join) to handle missing/duplicate slashes?